server/src/main/java/com/objectcomputing/checkins/services/agenda_item/AgendaItemController.java

```diff
@@ -43,18 +60,61 @@ public AgendaItemController(AgendaItemServices agendaItemServices) {
                 .body(error);
     }
 
-    /**	   
-     * Create and Save a new agenda item
+    @Error(exception = AgendaItemNotFoundException.class)
+    public HttpResponse<?> handleNotFound(HttpRequest<?> request, AgendaItemNotFoundException e) {
+        JsonError error = new JsonError(e.getMessage())
+                .link(Link.SELF, Link.of(request.getUri()));
+
+        return HttpResponse.<JsonError>notFound()
+                .body(error);
+    }
+
+    @Error(exception = AgendaItemsBulkLoadException.class)
+    public HttpResponse<?> handleBulkLoadException(HttpRequest<?> request, AgendaItemsBulkLoadException e) {
+        JsonError error = new JsonError(e.getMessage())
+                .link(Link.SELF, Link.of(request.getUri()));
+
+        return HttpResponse.<JsonError>badRequest()
+                .body(error);
+    }
+
+    @Error(exception = CompositeException.class)
+    public HttpResponse<?> handleRxException(HttpRequest<?> request, CompositeException e) {
+        LOG.info("OH NO COMPOSITES");
```

Comment:
Logging can be disabled / removed if it can slow down app.

Reply:
This is removed.